syntax = "proto2";
import "serverinfo_game.proto";
import "game_event_container.proto";

message GameReplay {
    optional uint64 replay_id = 1;
    optional ServerInfo_Game game_info = 2;
    repeated GameEventContainer event_list = 3;
    optional uint32 duration_seconds = 4;
}
